app/Http/Controllers/AdminSettingsController.php

```diff
@@ -27,14 +27,20 @@ public function __construct(AdminSettingsContract $adminSettingsContract)
         $this->adminSettingsUtility = $adminSettingsContract;
     }
 
+    private function authorizeUser()
+    {
+        $user = Auth::user();
+        $user->isAdmin();
+        return $user->organization_id;
+    }
+
     /**
      * Return current state of Organization Settings 
      * AND return all() Pay Period , time calc(NEXT ITERATION)
      * */
     public function getOrganizationSettings()
     {
-        $this->authorize('isAdmin', User::class);
-        $organizationId  = (Auth::user())->organization_id;
+        $organizationId = $this->authorizeUser();
```

Comment:
the naming here is kinda confusing

could probably just do a if check to see if the user is Admin, less lines and more read-able

Reply:
Look at the private method, I am doing this because we would be repeating code. 
The if check is happening within the user authorization trait file i created

Reply:
I agree the naming is confusing. your private method is named `authorizeUser` but the function is essentially `getOrgIdOfAuthorizedUser`

Reply:
this some c shit, like 
int fd[2];
int ret = pipe(fd);



what i mean is its cryptic fam